packages/rpc/test/chainStorageWatcher.test.ts

```diff
@@ -90,7 +90,7 @@ describe('makeAgoricChainStorageWatcher', () => {
   });
 
   it('can handle unserialized values', async () => {
-    const expected = 126560000000;
+    const expected = 'abc123';
```

Comment:
is this the only thing we need to test to have confidence?

also it's a little funny to start out with `expected`. consider `originaalValue` as the simple value being passed around . Or `value` and qualify the one come from chain storage.

Reply:
The difference is that `JSON.parse('abc123')`  throws while `JSON.parse(126560000000)` does not, so this test fails without the fix and passes with the fix. I just updated the test util to not stringify the value either to be sure. We could add a bunch of different cases with different data but I don't know if it's necessary when the distinction is: can it be parsed as JSON or not.

The `expected` thing is used everywhere in the test file, I'm not sold on `originalValue` because it could be taken to imply the value changed, which we have a specific test for, I'd rather just keep as is.